docs/schema-loader-import.md

```diff
@@ -131,20 +131,20 @@ Data types not listed in the above are not supported. The following are some com
 
 </div>
 
-<div id="PostgreSQL" class="tabcontent" markdown="1">
-
-| PostgreSQL        | ScalarDB | Notes                 |
-|-------------------|----------|-----------------------|
-| bigint            | BIGINT   | [*1](#warn-data-size) |
-| boolean           | BOOLEAN  |                       |
-| bytea             | BLOB     |                       |
-| character         | TEXT     | [*2](#warn-data-size) |
-| character varying | TEXT     | [*2](#warn-data-size) |
-| double precision  | DOUBLE   |                       |
-| integer           | INT      |                       |
-| real              | FLOAT    |                       |
-| smallint          | INT      | [*2](#warn-data-size) |
-| text              | TEXT     |                       |
+<div id="PostgreSQL_YugabyteDB" class="tabcontent" markdown="1">
+
+| PostgreSQL/YugabyteDB | ScalarDB | Notes                 |
+|-----------------------|----------|-----------------------|
+| bigint                | BIGINT   | [*1](#warn-data-size) |
```

Comment:
I don't think this is specific to this PR, but the links to `#warn-data-size` don't seem to work. Is there a section or footnote within this document that this link should point to?

Reply:
Good catch. The link seems to try to point https://github.com/scalar-labs/scalardb/blob/master/docs/schema-loader-import.md?plain=1#L272, but GitHub markdown automatically adds a prefix `user-content-`. So, it should be `user-content-warn-data-size`?

Reply:
Oooh, I see. I tried adding the footnotes within an admonition like you've done here, but doing so doesn't seem to be supported in Docusaurus. I'm guessing it has to do with the admonition component in Docusaurus, which isn't in HTML like the one in Jekyll is. 

> [!NOTE]
>
> I'm guessing the reasoning behind that is footnotes are their own section, so they shouldn't need to be inside an admonition. Books, academic papers, and such list their footnotes at the bottom of the pages as well, so Docusaurus is probably just following that standard.

Instead, how about if we use the standard [footnote feature in Markdown](https://www.markdownguide.org/extended-syntax/#footnotes)? This make a "Footnote" section at the end of the doc that users can refer to.[^1]

[^1]: This is an example of a footnote in Markdown.

Reply:
@josh-wong Thanks for the investigation. The footnote feature looks good. But, the content in the warning block is important and it should be still in the warning block more than in footnotes to keep them eye-catching?

I noticed the warning block is located below all the links pointing the block. I guess users can find the warning block without any links. So, how about replacing the links with just a comment like `See the below warning block (#2)`?

Reply:
BTW, I think we can fix the issue in a separate PR later.

Reply:
@komamitsu 
> I noticed the warning block is located below all the links pointing the block. I guess users can find the warning block without any links. So, how about replacing the links with just a comment like See the below warning block (#2)?

I'm OK with this method 👍